### ISSUE TITLE
Why does PSNR keep dropping when training a SRResnet with DIV2K dataset?

### ISSUE THREAD

**Issue by 1conglin** [Lornatang/SRGAN-PyTorch#95]
I trained the SRResnet with DIV2K training dataset( 800 images) and test set was DIV2K validation dataset(100 images)  , croppped each image to 512*512 with step 256, and GT size was set to 256*256, train SRResnet 90 epoches with batch size of 16.  But as the training progressed, PSNR kept dropping. Here are tensorboard logger figure:
![PSNR-dropping](https://github.com/user-attachments/assets/ddd2369b-245d-4069-abac-571edaed8589)
why did this happened?

**Comment by Lornatang** [comment]
The Loss curve shows that the model trained normally
try set lr to half

**Comment by 1conglin** [comment]
> The Loss curve shows that the model trained normally try set lr to half

It didn't work. I changed lr from 0.001 to 0.0005, but PSNR still kept dropping as training progressed.
![image](https://github.com/user-attachments/assets/f895fdaf-5399-483a-b1bf-5344d32aed4f)
Could you please tell me if there are any other possible reasons?

**Comment by 1conglin** [comment]
> The Loss curve shows that the model trained normally try set lr to half

I have found the reason, the train lr is different from test lr. train lr is generated from a complex function , but my test lr is just bicubic downsampled from test hr. I changed the train lr to bicubic downsamppling, then the problem resolved:
![image](https://github.com/user-attachments/assets/f8aea28f-2612-4a29-b950-16a1db1fbf6c)
![image](https://github.com/user-attachments/assets/c1e6737c-d3aa-4eb5-bd0c-e7c5bb0b0b0f)
Thank you for your attention and suggestion.
